entities/speedboost.lua

```diff
@@ -0,0 +1,22 @@
+
+function boost(player, speedBoost)
```

Comment:
This is a global variable. Try `local boost = function(...`

Reply:
Done